node/risk-app/server/routes/risk.js

```diff
@@ -43,7 +42,7 @@ module.exports = {
           return h.redirect('/england-only')
         }
         const backLinkUri = defineBackLink(path, address.postcode)
-        const htmlFile = config.riskPageFlag ? 'risk-flagged' : 'risk'
+        const htmlFile = 'risk'
```

Comment:
I would just remove this variable completely and hard code the file name into the view handler, as that's what's done in other routes.